model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -381,7 +373,7 @@ object ArrowInliner extends Logging {
       _ <- Arrows[S].purge
```

Comment:
`callArrowRet` introduces `Arrows[S].scope`, resolves some arrows and then calls `inline`
`inline` calls `prelude`
And the first thing `prelude` does is `Arrows[S].purge` once again.

Could this be optimized?

Reply:
Also lets move `Arrows[S].scope` and `Exports[S].scope` closer to each other?